prow/config/plugins.yaml

```diff
@@ -118,6 +118,17 @@ blockades:
       This PR modifies the files under the docs folder and requires the docs team to follow up on the PR.
 
       /label docs
+  - repos:
+      - pingcap/tidb
+    blockregexps:
+      - ^sessionctx/variable/
+      - ^config/config.toml.example$
+    explanation: |
+      This PR modifies significant config files which require review from domain experts from anyone of
+      
+      @dongyuusa, @bb7133, @XuHuaiyu, @cfzjywxk, or @morgo
```

Comment:
I have a question on this list as commented in the discussion thread[1], please do not merge yet.

[1] https://internals.tidb.io/t/topic/374/9

Reply:
👌

Reply:
@Mini256 according to the discussion, @qw4990 is proposed to switch @dongyuusa .

Please also update your work on the thread so that the audiences can know you're doing the work.